res/TensorFlowLiteRecipes/TransposeConv_003/test.rule

```diff
@@ -0,0 +1,5 @@
+# To check if CircleShape op is folded as constant.
+
+RULE    "VERIFY_FILE_FORMAT"      $(verify_file_format) '=' 1
+
+RULE    "NO_Shape"      $(op_count Shape) '=' 0
```

Comment:
I'm not sure whether it is enough... 🤔

Reply:
It would be better to make a simpler recipe. The current recipe contains irrelevant operators.

Reply:
I think it's enough. If you want to add more conditions, you may check other operators (not rehsape) are not removed.

Reply:
> It would be better to make a simpler recipe. The current recipe contains many irrelevant operators.

I plan to use this recipe to test for `FoldStridedSlice` and `FoldPack`

Or is it better to make independant recipe..? 🤔